charts/hybridnet/values.yaml

```diff
@@ -24,7 +24,10 @@ initNetworkConfig:
   preferVlanInterfaces: ""
   preferBGPInterfaces: ""
 
-advancedSettings:
+  # -- Whether the "init" Subnet and Network should skip the validation of webhook. true or false
+  skipValidation: false
```

Comment:
Because it might fail if the configuration of the webhook pod is changed when using `helm upgrade` to change the configuration of a one-master cluster.

If nothing have changed for `init` Network/Subnet, you can set this `skipValidation` flag `true` to execute `helm upgrade` commands,  then webhook pod will not cause a failure.

Reply:
Sure, `skipValidation` is removed and `init` Network/Subnet will only be created during `helm install`.

Then I delete one of the `objectSelector` fields of `hybridnet-validating-webhook`, which is used to ignore the validation of some networking/multicluster CR, since it look useless.